TodoApp/TodoApp.Api/App_Start/WebApiConfig.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System.Web.Http;
+using System.Web.Http.Routing;
+using Microsoft.Web.Http.Routing;
+
+namespace TodoApp.Api
+{
+    public static class WebApiConfig
+    {
+        public static void Register(HttpConfiguration config)
+        {
+            // Web API configuration and services
+
+            // Web API routes
+            var versionConstraintResolver = new DefaultInlineConstraintResolver()
```

Comment:
The `()` are redundant (basically remove any grayed code that clutters your reading experience :)) [applies to all solution, though, I noticed no other occurence]